packages/shared-components/src/components/FileLoader/FileLoader.js

```diff
@@ -59,7 +59,7 @@ export default class FileLoader extends React.Component {
       return (
         <DropArea>
           <Icon name="file" />
-          <Preview>Drop a file here, or click to browse.</Preview>
+          <Preview><span>DROP A FILE HERE, OR </span><span>CLICK TO BROWSE</span></Preview>
```

Comment:
Does anything special happen if the user clicks on the `CLICK TO BROWSE` part vs the `DROP A FILE HERE` part?

Reply:
no, according to Shaun the behavior is the same. All he wants is the underlined and different color of text.